scripts/ci/install_closure_deps.sh

```diff
@@ -7,25 +7,37 @@ declare -r CLANG_VERSION="3.7.1"
 declare -r CLANG_BUILD="clang+llvm-${CLANG_VERSION}-x86_64-linux-gnu-ubuntu-14.04"
 declare -r CLANG_TAR="${CLANG_BUILD}.tar.xz"
 declare -r CLANG_URL="http://llvm.org/releases/${CLANG_VERSION}/${CLANG_TAR}"
-declare -r CLOSURE_COMPILER_REPO="https://oss.sonatype.org/content/repositories/snapshots/com/google/javascript/closure-compiler/1.0-SNAPSHOT/"
 
 set -ex
 
 cd ..
 
+# Fetches Closure Compiler components from oss.sonatype.org
+fetch () {
+  local name="${1%-1.0-SNAPSHOT.jar}"
+  [ "$name" != "$1" ]
```

Comment:
It's an assertion.  If they're equal then the suffix-removal above didn't work.

That said, I could just as well leave off the `-1.0-SNAPSHOT.jar` from the argument and then not need to worry about that possibility.